Add task solution

davinnchii opened the pull request.

Denys14Samoilenko reviewed (approved):
Good Job 🔥 nothing much to add :)
some comment from my side, you can formatting a litlebit your code: 
` const formattedSource = sourceString.split(';') `


![Screenshot_2](https://github.com/mate-academy/js_style-to-object/assets/91065994/9d03ca50-8637-4652-909d-8c3a7f87bf32)

davinnchii commented:
> Good Job 🔥 nothing much to add :) some comment from my side, you can formatting a litlebit your code: `const formattedSource = sourceString.split(';')`
> 
> ![Screenshot_2](https://user-images.githubusercontent.com/91065994/259149484-9d03ca50-8637-4652-909d-8c3a7f87bf32.png)
Thanks
I did, but instead of `styles` I have like `sourceString.split(';')`. Making another variable only to split the parameter seems useless for me, cuz right after I split my string into array, I immediately iterate it, so.

dffUqp reviewed (changes requested):
LGTM but needs a few fixes:smiley:

roman-mirzoian reviewed (approved):
Well done 👍
